docs/source/en/model_doc/albert.mdx

```diff
@@ -67,104 +110,84 @@ This model was contributed by [lysandre](https://huggingface.co/lysandre). This
 
 ## AlbertModel
 
-[[autodoc]] AlbertModel
-    - forward
+[[autodoc]] AlbertModel - forward
```

Comment:
You can leave this alone as well and allow the `forward` method to be listed under the `AlbertModel` object. Same comment applies to all the other objects changed below :)